src/Resources/views/base_layout.html.twig

```diff
@@ -50,15 +50,21 @@ file that was distributed with this source code.
         {% block sonata_page_top_bar %}
             {% block page_top_bar %} {# Deprecated block #}
                 {% if sonata_page.isEditor or ( app.user and is_granted('ROLE_PREVIOUS_ADMIN') ) %}
-
+                    {#
```

Comment:
I think we should add this comment in every template where we can not use `admin.id()`.

Reply:
To be honest, I have no idea where is that. 

In 
https://github.com/sonata-project/SonataPageBundle/blob/78ae79486e5caf849f241bd14d97040106a853d5/src/Resources/views/Block/block_pagelist.html.twig#L24

I've changed it to `generateObjectUrl` because they come from 

https://github.com/sonata-project/SonataPageBundle/blob/78ae79486e5caf849f241bd14d97040106a853d5/src/Block/PageListBlockService.php#L85-L97
and they are actual "pages".